sdk/schemaregistry/Azure.Data.SchemaRegistry/src/Avro/SchemaRegistryAvroObjectSerializerOptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Data.SchemaRegistry.Avro
+{
+    /// <summary>
+    /// Options for <see cref="SchemaRegistryAvroObjectSerializer"/>.
+    /// </summary>
+    public class SchemaRegistryAvroObjectSerializerOptions
+    {
+        /// <summary>
+        /// When true, automatically registers the provided schema with the SchemaRegistry during serialization.
```

Comment:
Should we indicate what the default behavior is?

Also, I think we usually say "Gets or sets" at the beginning of get/set property docs.

Reply:
Fixed.